### PR TITLE
Refactor dir struct

### PR DESCRIPTION
**Pull request by eborysko-harvard**
[SKCP-26] Refactored project to just contain plugin assets. See iOS and Android client libraries for SlackClient Code.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/InternationalMixMob/CSCI-E-71_IMM_Slacker_Cordova_Plugin/pull/5" target="_blank">ReviewNinja

**Comment by eborysko-harvard** [comment]
Org'd the project and separated Client and Plugin files.

File: src/android/Slacker.java [hunk 1]
```diff
@@ -1,11 +1,11 @@
-package edu.cscie71.imm.app.slacker;
+package edu.cscie71.imm.slacker.plugin;
 
 import org.apache.cordova.CordovaPlugin;
 import org.apache.cordova.CallbackContext;
 import org.json.JSONArray;
 import org.json.JSONException;
 
-public class Slack extends CordovaPlugin {
+public class Slacker extends CordovaPlugin {
```
Comment (frederickjansen) [inline src/android/Slacker.java:8]: Should be renamed to SlackerPlugin according to plugin.xml

Comment (eborysko-harvard) [inline src/android/Slacker.java:8]: Good catch, I'll revert

File: plugin.xml [hunk 2]
```diff
@@ -9,10 +19,33 @@
   <platform name="android">
     <config-file target="res/xml/config.xml" parent="/*">
       <feature name="Slack">
-        <param name="android-package" value="edu.cscie71.imm.app.slacker.client.Slack"/>
+        <param name="android-package" value="edu.cscie71.imm.slacker.plugin.Slacker"/>
       </feature>
     </config-file>
-    <source-file src="src/android/Slack.java" target-dir="src/edu/cscie71/imm/app/slacker/Slack" />
```
Comment (frederickjansen) [inline plugin.xml:15]: This needs a rename to Slacker now as well

Comment (frederickjansen) [inline plugin.xml:15]: !fixed

File: src/android/Slacker.java (hunk 1 above)
Comment (frederickjansen) [inline src/android/Slacker.java:8]: !fixed

**Pull request merged**